inc/db.function.php

```diff
@@ -1736,6 +1736,10 @@ function getEntitiesRestrictRequest($separator = "AND", $table = "", $field = ""
    if (is_array($value)) {
       $query .= " IN ('" . implode("','", $value) . "') ";
    } else {
+      if (strlen($value) == 0 && !isset($_SESSION['glpiactiveentities_string'])) {
+         //set root entity if not set
+         $value = 0;
+      }
       if (strlen($value) == 0) {
```

Comment:
regarding the original issue, if ``$_SESSION['glpiactiveentities_string']`` is not set, we still pass in this line and the notice will still trigger.

Maybe we could replace ``if`` by ``else if`` or move the ``!isset(...)`` part into the following block.